File: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples-js/GetLinkedBackendsForStaticSite.js
const { WebSiteManagementClient } = require("@azure/arm-appservice");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Returns details of all backends linked to a static site
 *
 * @summary Returns details of all backends linked to a static site
 * x-ms-original-file: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples/GetLinkedBackendsForStaticSite.json
 */
async function getDetailsOfTheLinkedBackendsRegisteredWithAStaticSite() {
  const subscriptionId = "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345";
  const resourceGroupName = "rg";
  const name = "testStaticSite0";
  const credential = new DefaultAzureCredential();
  const client = new WebSiteManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.staticSites.listLinkedBackends(resourceGroupName, name)) {
    resArray.push(item);
  }
  console.log(resArray);
}

getDetailsOfTheLinkedBackendsRegisteredWithAStaticSite().catch(console.error);

File: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples-java/GetLinkedBackendsForStaticSite.java
import com.azure.core.util.Context;

/** Samples for StaticSites GetLinkedBackends. */
public final class Main {
    /*
     * x-ms-original-file: specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/examples/GetLinkedBackendsForStaticSite.json
     */
    /**
     * Sample code: Get details of the linked backends registered with a static site.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void getDetailsOfTheLinkedBackendsRegisteredWithAStaticSite(
        com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .webApps()
            .manager()
            .serviceClient()
            .getStaticSites()
            .getLinkedBackends("rg", "testStaticSite0", Context.NONE);
    }
}
